homeassistant/components/modbus/sensor.py

```diff
@@ -263,6 +267,21 @@ def available(self) -> bool:
         """Return True if entity is available."""
         return self._available
 
+    def _swap_registers(self, registers):
```

Comment:
Is there no helper in the library to do this?

Reply:
I have not been able to find one. The socket library provides part of such a function, but not all we need. I tried to google, and this was the most elegant solution I could find.

Reply:
Ok. I think we're crossing into the protocol details here, so it would be nice to offload to the library. But it could be ok for now.

Reply:
I agree it is in the grey level. I will make a PR against pymodbus and see if they are willing to accept in for one of their future versions.